### ISSUE TITLE
Support side-by-side diff

### ISSUE THREAD

**Issue by sgraham** [esprehn/chromium-codereview#2]
I'm awful at reading unified +- diffs, I'd love a side-by-side diff option.

**Comment by esprehn** [comment]
Yeah, that's on the roadmap, Ojan really wants it too. Unfortunately it's a lot of effort so it probably won't be for a while.

**Comment by dalecurtis** [comment]
:+1: , sadly unusable for my lowly mind without it :(
